test/tests/optimizationreporter/objective_gradient_minimize/bc_load_linearFunction/master.i

```diff
@@ -86,8 +86,8 @@
     [fromForward]
       type = MultiAppReporterTransfer
       from_multi_app = forward
-      from_reporters = 'data_pt/temperature data_pt/temperature'
-      to_reporters = 'OptimizationReporter/simulation_values receiver/measured'
+      from_reporters = 'data_pt/temperature'
+      to_reporters = 'OptimizationReporter/simulation_values'
```

Comment:
The changes to these input seem unrelated to the purpose of this PR. What is the purpose?

Reply:
It isn't related to this PR.  I'm trying to clean up the input files so that transfers no longer write their data to a constant reporter on the main app.  All of the information written to the constant reporter is available in the OptimizationReporter so now I'm changing the csvdiffs to use that file and getting rid of the constant reporter.

Reply:
I also changed the executioners to use Newton and line_search=none